trie.py

```diff
@@ -0,0 +1,73 @@
+# https://www.codingninjas.com/studio/problems/implement-trie_631356?leftPanelTabValue=PROBLEM
+from sys import stdin, setrecursionlimit
+setrecursionlimit(10**7)
+
+
+class Node:
+    def __init__(self):
+        self.links = [None] * 26
+        self.flag = False
+
+class Trie:
+    def __init__(self):
+        self.root = Node()
+
+    def insert(self, string):
+        node = self.root
+        for char in string:
+            pos = ord(char) - ord('a')
+            if not node.links[pos]:
+                node.links[pos] = Node()
+            node = node.links[pos]
+        node.flag = True
+
+    
+    def search(self, word):
+        node = self.root
+        for char in word:
+            pos = ord(char) - ord('a')
+            if not node.links[pos]:
+                return False
+            node = node.links[pos]
+
+        if node.flag == False:
+            return False
+        return True
+
+        
+    def startWith(self, prefix):
+        node = self.root
+        for char in prefix:
+            pos = ord(char) - ord('a')
+            if not node.links[pos]:
+                return False
+            node = node.links[pos]
+        return True
```

Comment:
this also can be simplified using `is not None` check

Reply:
I feel the existing one is fine since we need to exit if link to any of the char is not there